packages/components/src/gravatar-logo/index.tsx

```diff
@@ -0,0 +1,24 @@
+import classNames from 'classnames';
+
+type Props = {
+	className?: string;
+	size?: number;
+};
+
+export function GravatarLogo( { className, size = 32 }: Props ) {
+	const classes = classNames( 'gravatar-logo', className );
+
+	return (
+		<svg className={ classes } height={ size } viewBox="0 0 146 27">
```

Comment:
BTW I wanted to use `SVG` & `Path` primitives from `@wordpress/component` but they caused errors that I couldn't resolve:

```
/Users/mikael/Documents/Work/Automattic/Calypso/wp-calypso/node_modules/progress-event/index.js:26
'function' === typeof document.createEvent ? function ProgressEvent (type, props) {
                      ^
ReferenceError: document is not defined
    at Object.<anonymous> (/Users/mikael/Documents/Work/Automattic/Calypso/wp-calypso/node_modules/progress-event/index.js:26:23)
    at Module._compile (node:internal/modules/cjs/loader:1155:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1209:10)
    at Module.load (node:internal/modules/cjs/loader:1033:32)
    at Function.Module._load (node:internal/modules/cjs/loader:868:12)
    at Module.require (node:internal/modules/cjs/loader:1057:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object../client/layout/logged-out.jsx (/Users/mikael/Documents/Work/Automattic/Calypso/wp-calypso/build/webpack:/wp-calypso/external commonjs "progress-event":1:35)
    at __webpack_require__ (/Users/mikael/Documents/Work/Automattic/Calypso/wp-calypso/build/webpack:/wp-calypso/webpack/bootstrap:19:1)
    at Object../client/controller/index.node.js (/Users/mikael/Documents/Work/Automattic/Calypso/wp-calypso/build/server.js:9413:18)
```

@Automattic/team-calypso-frameworks do you know what's going on with that?

Reply:
From what you shared it seems like those components aren't supported in the SSR context, and the `DefaultOauthClientMasterbar` component is used in SSR context, for the logged-out masterbar.

Reply:
Although, this seems weird because I don't see anything in common between `progress-event` and those components, and the `@wordpress/components` package in general. Would you mind pushing a branch with what you tried so we could take a look?